src/flows/pipes/RawFluxEditor/FunctionsList/DynamicFunctionsList.tsx

```diff
@@ -24,19 +24,44 @@ const sortFuncs = (a, b) => {
     return a.package.toLowerCase() < b.package.toLowerCase() ? -1 : 1
   }
 }
+const hoveredFunctions = new Set<string>()
+let recordedFunctions = new Set<string>()
 
 const DynamicFunctionsList: FC<Props> = ({onSelect}) => {
+  const [eventSearchTerm, setEventSearchTerm] = useState('')
+  const [termRecorded, setTermRecorded] = useState('')
+  const [tooltipPopup, setTooltipPopup] = useState(false)
+  const [hoverdFunction, setHoverdFunction] = useState('')
+
   const dispatch = useDispatch()
   const fluxFunctions = useSelector(getAllFluxFunctions)
+
   useEffect(() => {
     if (fluxFunctions.length === 0) {
       dispatch(getFluxPackages())
     }
   }, [])
 
+  useEffect(() => {
+    if (tooltipPopup && eventSearchTerm !== termRecorded) {
+      hoveredFunctions.clear()
+      event('flux.function.searched', {searchTerm: eventSearchTerm})
+      setTermRecorded(eventSearchTerm)
+    }
+    setTooltipPopup(false)
+    if (tooltipPopup) {
+      hoveredFunctions.add(hoverdFunction)
+
+      if (recordedFunctions.size !== hoveredFunctions.size) {
+        event('hovered.flux.function', hoveredFunctions)
```

Comment:
Something like this:
```
event('hovered.flux.function', {hoveredFunction, searchTerm: eventSearchTerm})
```

but make sure it's at the moment of hover

Reply:
also, the name of the event might need tweaking. maybe something like `flux.function.hover`

Reply:
it's firing right on hover, so we're good there. But adding eventSearchTerm there would mean we will be sending an empty string. If that's not an issue, i can easily add it. My thought process was; I have a code that handles firing event for search term, so we dont need it here.